app/sidekiq/lighthouse/submit_benefits_intake_claim.rb

```diff
@@ -0,0 +1,141 @@
+        :error,
+        "Failed all retries on CentralMail::SubmitBenefitsIntakeClaim, last error: #{msg['error_message']}"
+      )
+      StatsD.increment("#{STATSD_KEY_PREFIX}.exhausted")
+    end
+
+    # rubocop:disable Metrics/MethodLength
+    def perform(saved_claim_id)
+      @claim = SavedClaim.find(saved_claim_id)
+      @pdf_path = process_record(@claim)
+      @attachment_paths = @claim.persistent_attachments.map do |record|
+        process_record(record)
+      end
+
+      @lighthouse_service = BenefitsIntakeService::Service.new(with_upload_location: true)
+      create_form_submission_attempt(@lighthouse_service.uuid)
+
+      payload = {
+        upload_url: @lighthouse_service.location,
+        file: split_file_and_path(@pdf_path),
+        metadata: generate_metadata.to_json,
+        attachments: @attachment_paths.map(&method(:split_file_and_path))
+      }
+
+      response = @lighthouse_service.upload_doc(**payload)
+
+      if response.success?
+        log_message_to_sentry('CentralMail::SubmitSavedClaimJob succeeded', :info, generate_sentry_details)
+        @claim.send_confirmation_email if @claim.respond_to?(:send_confirmation_email)
+      else
+        raise BenefitsIntakeClaimError, response.body
+      end
+    rescue => e
+      log_message_to_sentry('CentralMail::SubmitBenefitsIntakeClaim failed, retrying...', :warn,
+                            generate_sentry_details(e))
+      raise
+    ensure
+      cleanup_file_paths
+    end
+
+    # rubocop:enable Metrics/MethodLength
+    def generate_metadata
+      form = @claim.parsed_form
+      veteran_full_name = form['veteranFullName']
+      address = form['claimantAddress'] || form['veteranAddress']
+
+      metadata = {
+        'veteranFirstName' => veteran_full_name['first'],
+        'veteranLastName' => veteran_full_name['last'],
+        'fileNumber' => form['vaFileNumber'] || form['veteranSocialSecurityNumber'],
+        'zipCode' => address['country'] == 'USA' ? address['postalCode'] : FOREIGN_POSTALCODE,
+        'source' => "#{@claim.class} va.gov",
+        'docType' => @claim.form_id,
+        'businessLine' => business_line
+      }
+
+      SimpleFormsApiSubmission::MetadataValidator.validate(metadata)
+    end
+
+    def process_record(record)
+      pdf_path = record.to_pdf
+      stamped_path = CentralMail::DatestampPdf.new(pdf_path).run(text: 'VA.GOV', x: 5, y: 5)
+      CentralMail::DatestampPdf.new(stamped_path).run(
+        text: 'FDC Reviewed - va.gov Submission',
+        x: 429,
+        y: 770,
+        text_only: true
+      )
+    end
+
+    def split_file_and_path(path)
+      { file: path, file_name: path.split('/').last }
+    end
+
+    def business_line
+      case @claim.class
+      when SavedClaim::VeteranReadinessEmploymentClaim
+        'VRE'
+      when SavedClaim::EducationCareerCounselingClaim
+        'EDU'
+      when SavedClaim::Burial
+        'NCA'
+      else
+        ''
+      end
+    end
```

Comment:
Could you potentially create a `business_line` method on each of these classes that would return 'VRE' 'EDU', 'NCA', etc via the duck typing pattern? Kinda a nit, but would allow us to clean up this conditional and push it to a method on the class.

Reply:
Yeah, that makes sense. I'll implement those as well as in the base class